tests/sentry/api/endpoints/test_project_transaction_threshold.py

```diff
@@ -0,0 +1,172 @@
+from django.urls import reverse
+
+from sentry.models.transaction_threshold import ProjectTransactionThreshold
+from sentry.testutils import APITestCase
+
+
+class ProjectTransactionThresholdTest(APITestCase):
+    feature_name = "organizations:project-transaction-threshold"
+
+    def test_get(self):
+        self.login_as(user=self.user)
+        project = self.create_project()
```

Comment:
You can probably move this and the `reverse(...)` into a setup method so you don't have to repeat it

Reply:
yeah lol oops